src/coreclr/tools/aot/DependencyGraphViewer/DependencyGraphsForm.cs

```diff
@@ -5,19 +5,26 @@
 using System.Collections.Generic;
 using System.ComponentModel;
 using System.Data;
+using System.Diagnostics;
 using System.Drawing;
+using System.IO;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 using System.Windows.Forms;
+using Microsoft.Diagnostics.Tracing.Session;
 
 namespace DependencyLogViewer
 {
     public partial class DependencyGraphs : Form
     {
-        public DependencyGraphs()
+        private int fileCount = -1;
```

Comment:
[Style] Private local fields should have a `_` prefix.

Reply:
noted